src/external/governance/Governor_v1.sol

```diff
@@ -174,11 +177,10 @@ contract Governor_v1 is ERC165, IGovernor_v1, AccessControlUpgradeable {
 
     /// @inheritdoc IGovernor_v1
     function setFeeManager(address newFeeManager)
-        external
+        public
```

Comment:
Why not keep it external?

Reply:
Is there a need for it to be called from within this contract?

Reply:
I did that before the internal function
Will change it